src/track/track.cpp

```diff
@@ -25,8 +25,17 @@ SecurityTokenPointer openSecurityToken(
     }
 }
 
+struct TrackDeleter {
```

Comment:
Why we need a functor here, and not just? 

void trackDeleter(Track* pTrack) {
       if (pTrack != nullptr) {
           pTrack->deleteLater();
       }
 }